pkg/k8shandler/collection.go

```diff
@@ -51,6 +53,42 @@ func CreateOrUpdateCollection(cluster *logging.ClusterLogging) (err error) {
 			}
 		}
 	}
+	if cluster.Spec.Collection.LogCollection.Type == logging.LogCollectionTypeRsyslog {
+		if err = createOrUpdateRsyslogServiceAccount(cluster); err != nil {
+			return
+		}
+
+		if err = createOrUpdateCollectionPriorityClass(cluster); err != nil {
```

Comment:
since this is common for both fluentd and rsyslog, can we just move it up a level out of both of their if statements?